### ISSUE TITLE
can not save site access token

### ISSUE THREAD

**Issue by billzhuang** [buunguyen/octotree#168]
use network monitor, found it return a 404, seems github api already updated.

**Comment by buunguyen** [comment]
Hmm, seems to still work fine. Please provide more info to help reproduce. Thanks.

**Comment by schwarzmx** [comment]
I'm not sure if it's the same as @billzhuang's issue, but octotree asks me for my access token when I switch to a branch whose name has slashes like `feature/myfeature`. On chrome's console the call to the github api is incomplete thus returning a 404. I.e., what the request should be:

```
https://api.github.com/repos/:owner/:repo/git/trees/feature/myFeature?recursive=1
```

And what octotree constructs:

```
https://api.github.com/repos/:owner/:repo/git/trees/feature?recursive=1
```

Note the incomplete branch name.

**Comment by kidonng** [comment]
I encountered the same problem as @schwarzmx's as well, you can test it here: https://github.com/kidonng/swiss/blob/archive/xkcd-why

**Comment by buunguyen** [comment]
@kidonng thanks. GH just changed their DOM thus causes this error. Fixed in 5.2.1.
